zipline/finance/performance/tracker.py

```diff
@@ -171,6 +166,13 @@ def __init__(self, sim_params, env):
         self.account_needs_update = True
         self._account = None
 
+        self._perf_periods = [self.cumulative_performance,
+                              self.todays_performance]
+
+    @property
+    def perf_periods(self):
+        return self._perf_periods
```

Comment:
Is this wrapped in a property to prevent re-assignment (why is that a concern for this attribute)?  Wondering the perf impact.